### PR TITLE
Update comment of LB token

### PR DESCRIPTION
**Pull request by AspirinSJL**
Requested [here](https://github.com/grpc/grpc/issues/8363#issuecomment-360280024).

### CODE REVIEW

**Comment by AspirinSJL** [comment]
@dgquintas Please sync the change to grpc repo as part of #15728 once this PR is merged. Thanks!

**Comment by AspirinSJL** [comment]
@yetianx, requesting review from this comment because I still can't find your ID via search. :/

**Comment by carl-mastrangelo** [comment]
@AspirinSJL When this change is submitted, please make sure to vendor the new file to the other repos.

**Comment by dgquintas** [comment]
Using this version of the proto in core is not something we can currently do. See https://github.com/grpc/grpc/issues/15728#issuecomment-398925386

**Comment by AspirinSJL** [comment]
I see. The comment updating is not urgent in other repos. So I think we can update it here now, and sync the change whenever the issue @dgquintas pointed out is solved and we are ready to use this proto in core.

**Comment by AspirinSJL** [comment]
Can I get an approval for this PR so that we can include the update in [#15840](https://github.com/grpc/grpc/pull/15840)?

File: grpc/lb/v1/load_balancer.proto [hunk 1]
```diff
@@ -130,11 +130,13 @@ message Server {
...
-  // metadata. The token is used by the backend to verify the request and to
-  // allow the backend to report load to the gRPC LB system. The token is also
-  // used in client stats for reporting dropped calls.
+  // An opaque but printable token for load reporting. The client must include
+  // the token of the picked server into the initial metadata when it starts a
+  // call to that server. The token is used by the server to verify the request
+  // and to allow the server to report load to the gRPC LB system. The token is
+  // also used in client stats for reporting dropped calls.
+  //
+  // Its length can be variable but must be less than 50 bytes.
```
Comment (dgquintas) [inline grpc/lb/v1/load_balancer.proto]: nit: s/less/fewer.

**Comment by dgquintas** [comment]
Actually, no nit, my bad. LGTM

**Comment by dgquintas** [comment]
I've updated https://github.com/grpc/grpc/pull/15840 as well

**Pull request merged**